data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -204,15 +225,33 @@
     "viewing_hint": "CV",
     "range": "???"
   },
-  "contributor": [
-    "array of strings (CV)"
-  ],
-  "related_objects": [
-    {
-      "relationship_type": "CV",
-      "object": "ARK",
-      "position": "int",
-      "layer": "int"
-    }
-  ]
+  "related_objects": {
+    "has_content": [
+      {
+        "type": "CV",
+        "object": "ARK",
+        "position": "int"
+      }
+    ],
+    "part_of": [
```

Comment:
codicological units have both child text relationships ('has_content') and parent ms object relationship ('part_of').

For the 'part_of' I'm thinking it would be auto-populated and so not necessarily need the additional fields? (i.e., the layer and sequence position would be contained in the 'has_part' relationship from the ms object)